test/managedcluster/managedcluster.go

```diff
@@ -66,17 +76,18 @@ func GetProviderLabel(provider ProviderType) string {
 
 // GetUnstructured returns an unstructured ManagedCluster object based on the
 // provider and template.
-func GetUnstructured(templateName Template) *unstructured.Unstructured {
+func GetUnstructured(templateName Template, provider ProviderType) *unstructured.Unstructured {
```

Comment:
As specified above, we do not need `ProviderType` here anymore, its unused below anyways.  Rebase might have brought it back accidentally or something.  I'm surprised the linter didn't catch it's unused.